malware/system/ksp-doki-malware.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-doki-malware
+  namespace: multiubuntu # Change your namespace
+spec:
+  tags: ["DOKI","MALWARE"]
+  message: "Alert! Doki malware execution stopped"
+  selector:
+    matchLabels:
+      container: ubuntu-1 # Change your matchLabels
+  process:
+    severity: 3
+    matchPaths:
+    - path: /usr/bin/curl
```

Comment:
This seems to me that if I do `curl` it'll give me block log for Doki malware
- User can do curl for google as well
- Confidence factor it low

Reply:
This policy currently not a good approach protect against doki malware and testing is not completed in any environment.
Closing this PR.
cc: @raviknox